README.md

```diff
@@ -1 +1,83 @@
+ - Open settings.json and find `applicationPath` option under the `driverSettings` section of desired platform. Replace the value with full or relative path to your app, e.g. `./Resources/Apps/ApiDemos-debug.apk`.
+
+3. Ensure that [Appium server](https://appium.io) is set up at your machine where the code would be executed, and the address/port match to set in your `settings.json` in `remoteConnectionUrl` parameter.
+If the parameter `isRemote` in your settings.json is set to `false`, this means that AppiumDriverLocalService would be used to setup Appium server using Node.js. This option requires specific version of node.js to be preinstalled on your machine (Please read more [here](http://appium.io/docs/en/contributing-to-appium/appium-from-source/#nodejs) )
+
+4. (optional) Launch an application directly by calling `var application = AqualityServices.Application;`. 
+> Note: 
+If you don't start an Application directly, it would be started with the first call of any Aquality service or class requiring interacting with the Application.
+
+5. That's it! Now you are able work with Application via AqualityServices or via element services.
+Please take a look at our example tests [here](Aquality.Appium.Mobile/tests/Aquality.Appium.Mobile.Tests/Samples/)
+
+6. To interact with Application's forms and elements, we recommend following the Page/Screen Objects pattern. This approach is fully integrated into our package.
+To start with that, you will need to create a separate class for each window/form of your application, and inherit this class from the [AndroidScreen](Aquality.Appium.Mobile/src/Aquality.Appium.Mobile/Screens/AndroidScreen.cs) or [IOSScreen](Aquality.Appium.Mobile/src/Aquality.Appium.Mobile/Screens/IOSScreen.cs) respectively. 
+
+> We recommend to use separate Screen class for each form of your application. You can take advantage of inheritance and composition pattern. We also suggest not to mix app different platforms in single class: take advantage of interfaces instead, adding the default implementation to them if is needed.
+
+
+7. From the Screen Object perspective, each Screen consists of elements on it (e.g. Buttons, TextBox, Labels and so on). 
+To interact with elements, on your form class create fields of type IButton, ITextBox, ILabel, and initialize them using the `AqualityServices.ElementFactory`. Created elements have a various methods to interact with them. We recommend combining actions into a business-level methods:
+
+```csharp
+using Aquality.Appium.Mobile.Elements.Interfaces;
+using Aquality.Appium.Mobile.Screens;
+using OpenQA.Selenium;
+
+namespace Aquality.Appium.Mobile.Tests.Samples.Android.ApiDemosScreens
+{
+    public class InvokeSearchScreen : AndroidScreen
+    {
+
+        private readonly ITextBox searchTextBox;
+        private readonly IButton startSearchButton;
+        private readonly ILabel searchResultLabel;
```

Comment:
Could we use properties here instead of initialization in constructor?

Reply:
Yes we could, but we probably should not, because we will achieve different behavior.
When you store elements in private fields, you can cache them, this could speed up elements search and interaction.
When you use properties for them, you will get a new element each time, so the search for element will be started from the scratch. The 'properties' way would be recommended for Web, but for mobile Appium  driver and complex application's page source I suggest to use 'private fields' approach